modules/core/CMakeLists.txt

```diff
@@ -27,5 +27,9 @@ ocv_glob_module_sources(SOURCES "${OPENCV_MODULE_opencv_core_BINARY_DIR}/version
 ocv_module_include_directories(${the_module} ${ZLIB_INCLUDE_DIRS})
 ocv_create_module(${extra_libs})
 
+if(HAVE_VA)
+  ocv_target_link_libraries(opencv_core "${VA_EXTRA_LIBS}")
```

Comment:
Prefered way is on the line 4 of this file. Empty variable value in case of unavailable `VA` should be processed fine (like `OPENCL_LIBRARIES`).